### ISSUE TITLE
Crash when Localize Document

### ISSUE THREAD

**Issue by Omarkth** [mfractor/mfractor-feedback#165]
Visual Studio for Mac just hangs immediately after clicking on "Localize Document"

##Installation Information

```
--Platform Information--
Operating System: Mac
OS: Unix
OS Service Pack: 
OS Version: Unix 19.0.0.0
--MFractor Information--
Product Name: MFractor
Product Version: 3.9.1
Product Variant: Visual Studio Mac
Product Variant Version: 8.3.10.2
Product SKU: MFRACTOR-VS-MAC
Commit SHA: f6274730759bb9c31553729d38c1f6a5ef19a418
Revision: 3952
Branch: master
Date: 5/12/2019 10:25:17 PM (UTC)
Agent: Local
Installed extensions:
	
```

**Comment by matthewrdev** [comment]
Thanks for reporting this @Omarkth. I'll look into it.

**Comment by matthewrdev** [comment]
@Omarkth This has been fixed and will be available in MFractor 3.9.2 available later today.
